v3-sdk/minting-position/src/example/Example.tsx

```diff
@@ -0,0 +1,249 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import './Example.css'
+import { ethers } from 'ethers'
+import {
+  Pool,
+  computePoolAddress,
+  Position,
+  nearestUsableTick,
+  NonfungiblePositionManager,
+} from '@uniswap/v3-sdk'
+import { Percent } from '@uniswap/sdk-core'
+import { Environment, CurrentConfig } from '../config'
+import { getCurrencyBalance } from '../libs/balance'
+import { getPositionIds, getTokenTransferApprovals } from '../libs/positions'
+import {
+  connectBrowserExtensionWallet,
+  getProvider,
+  TransactionState,
+  sendTransaction,
+  getWalletAddress,
+} from '../libs/providers'
+import IUniswapV3PoolABI from '@uniswap/v3-core/artifacts/contracts/interfaces/IUniswapV3Pool.sol/IUniswapV3Pool.json'
+import {
+  POOL_FACTORY_CONTRACT_ADDRESS,
+  MAX_FEE_PER_GAS,
+  MAX_PRIORITY_FEE_PER_GAS,
+  NONFUNGIBLE_POSITION_MANAGER_CONTRACT_ADDRESS,
+} from '../libs/constants'
+
+const getPoolInfo = async (): Promise<{
+  token0: string
+  token1: string
+  fee: number
+  tickSpacing: number
+  sqrtPriceX96: ethers.BigNumber
+  liquidity: ethers.BigNumber
+  tick: number
```

Comment:
Time to break this into an interface? `PoolInfo`?

Reply:
I intentionally did not pull this out, to not complicate things too much but I guess I'm good with it.